search/views.py

```diff
@@ -4,21 +4,23 @@
 from iogt.settings.base import SEARCH_RESULTS_PER_PAGE
 from wagtail.search.models import Query
 
+from questionnaires.models import Survey, Quiz
+
 
 def search(request):
-    search_groups = (Article, Section)  # add models which should be searchable
+    search_groups = (Article, Section, Survey, Quiz)
 
     search_query = request.GET.get("query")
-    page = request.GET.get("page", 1)
+    page = int(request.GET.get("page", 1))
 
     results = {"search_query": search_query, "search_groups": {}}
 
     # Search
     for search_group in search_groups:
-        search_results_no = 0
+        search_results_amount = 0
```

Comment:
I think amount suggests a monetary notion to the variable. Can we revert it or make it search_results_count?

Reply:
ok, will be changed

Reply:
✅